aws/data_source_aws_subnet.go

```diff
@@ -39,6 +39,12 @@ func dataSourceAwsSubnet() *schema.Resource {
 				Computed: true,
 			},
 
+			"outpost_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Since this attribute cannot be used as an argument to limit the data source results, `Optional` should be removed. 👍 

```suggestion
```